doc/nuake.txt

```diff
@@ -52,6 +52,14 @@ let g:nuake_size = {0-1} (default 0.25)
 let g:nuake_per_tab = {0,1} (default 0)
     Enable the Nuake instance per tab page.
 
+                                                             *g:close_if_last_standing*
+let g:close_if_last_standing = {0,1} (default 1)
+    Close vim if the Nuake window is the last window.
```

Comment:
Because plugin is both for Neovim and Vim it may be better to write it as follow:

> Close the editor if the Nuake window is the last one.

Or something like that ;)

Reply:
I changed it to editor. I use Neovim actually :)